hugegraph-server/hugegraph-dist/src/assembly/static/bin/util.sh

```
@@ -279,17 +613,36 @@ function get_ip() {
+            mv -f -- "$tmp" "$dest"
+        else
+            rm -f -- "$tmp"
+            return 1
         fi
-        curl -L "${download_url}" -o "${path}/$(basename "${download_url}")"
     elif command_available "wget"; then
-        wget --help | grep -q '\--show-progress' && progress_opt="-q --show-progress" || progress_opt=""
-        wget "${download_url}" -P "${path}" $progress_opt
+        local progress_opt=""
+        wget --help | grep -q -- '--show-progress' && progress_opt="-q --show-progress"
+        if wget -- "${download_url}" -O "$tmp" $progress_opt; then
```

Shellcheck (0.11.0)

[info] Double quote to prevent globbing and word splitting.

(SC2086)